lib/home/view/widgets/navbar_header.dart

```diff
@@ -42,7 +44,11 @@ class NavHeader extends StatelessWidget {
               children: [
                 showSearchIcon
                     ? InkWell(
-                        onTap: () => print('Navigate to search scren'),
+                        onTap: () {
+                          context.pushRoute(
```

Comment:
Why does the search icon lead to the LatesActivities screen instead of the Search screen?

Reply:
I was just testing out the Nested navigation